src/storage/compactor/src/server.rs

```diff
@@ -216,7 +216,7 @@ pub async fn compactor_serve(
             Duration::from_millis(config.server.heartbeat_interval_ms as u64),
             vec![sstable_object_id_manager],
         ),
-        risingwave_storage::hummock::compactor::Compactor::start_compactor(
+        risingwave_storage::hummock::compactor::compactor_runner::start_compactor(
```

Comment:
how about move `start_compactor`  to  compactor mod ?


For me, the runner corresponds to an operation performed once. rather than a persistently running thread. `start_compactor` is an event_loop

Reply:
Make Sense to me.